### DISCUSSION TITLE
Husk in drone won't seek and attack human in mainsub

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15386] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/b24e4384-ca4f-4a92-949c-bc8245066fe7

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
https://github.com/user-attachments/assets/25df4465-7a0a-4c75-a5d9-2880a088cf33

**Comment by SoobGorl** [comment]
This looks to be an issue with creature pathfinding as a whole. Ticketed in [private branch as issue #7480](https://github.com/FakeFishGames/Barotrauma-development/issues/7480).
